content/applications/sales/sales/gelato.rst

```diff
@@ -4,4 +4,166 @@
 Gelato
 ======
 
-The documentation for Gelato will be added soon.
+Gelato is a global print-on-demand platform that integrates with Odoo to sync product catalogs and
+automate order fulfillment.
+
+Connecting Gelato's services with Odoo's **Sales** and **eCommerce** apps enables the following:
+
+- Sync Odoo sales orders with Gelato for automated order fulfillment
+- Create and manage Gelato products within Odoo; supports product variant and image sync
+- Configure delivery options in Odoo and receive order updates via webhooks.
+
+Configuration
+=============
+
+.. important::
+   The company information (Company Name and Billing address) in the Gelato account *must* match the
+   company information in the Odoo database in order for sales orders to be confirmed and sent to
+   Gelato for fulfillment.
+
+   .. image:: gelato/gelato-company.png
+      :alt: Company information in Gelato.
+
+   .. image:: gelato/odoo-company.png
+      :alt: Company information in Odoo.
+
+Configure API Keys and Webhooks in Gelato
+-----------------------------------------
+
+Before configuring the Gelato connector in Odoo, first obtain API credentials and webhooks from the
+Gelato account.
+
+API connectors enable Odoo **Sales** to send and receive data from Gelato for order processing,
+while webhooks provide real-time updates on order status and shipment tracking.
+
+API Key
+~~~~~~~
+
+An API Key is a unique authentication token that allows Odoo to securely communicate with Gelato’s
+API, enabling order transmission, status updates, and data synchronization.
+
+After logging into Gelato, click :icon:`fa-code` :guilabel:`Developer` in the left menu bar. From
+here, click on :guilabel:`API keys`. In the new page, click the :guilabel:`Add API Key` button to
+open a new API key form. Type in a name, then click :guilabel:`Create Key`.
+
+Copy the generated API key using :guilabel:`Copy to Clipboard`.
+
+.. image:: gelato/gelato-api-key.png
+   :alt: Newly generated API key in the Gelato platform.
+
+.. important::
+   Copy the API key and store on a notepad before leaving this page. Once the page is refreshed or
+   exited, the key will not be available to copy.
```

Comment:
Best to advise keeping API keys somewhere secure rather than a plain text notepad :)
```suggestion
   Copy the API key and store it somewhere safe and secure before leaving this page. Once the page
   is refreshed or exited, the key will not be available to copy.
```